cmd/tetra/getevents/io_reader_client.go

```diff
@@ -10,10 +10,10 @@ import (
 	"io"
 	"os"
 
-	v1 "github.com/cilium/hubble/pkg/api/v1"
-	hubbleFilters "github.com/cilium/hubble/pkg/filters"
 	"github.com/cilium/tetragon/api/v1/tetragon"
 	"github.com/cilium/tetragon/pkg/filters"
+	v1 "github.com/cilium/tetragon/pkg/oldhubble/api/v1"
```

Comment:
Could we please export it as  hubbleV1 like in other places?

Reply:
This patch just replaces:
> 	v1 "github.com/cilium/hubble/pkg/api/v1"

from the previous version.

Added a patch to use hubbleV1.